### PR TITLE
Spec: Add placeholder to Input.ChoiceSet

### PR DESCRIPTION
**Pull request by andrewleader**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/3199)

### CODE REVIEW

**Comment by nesalang** [comment]
thanks!

File: schemas/src/elements/inputs/Input.ChoiceSet.json [hunk 1]
```diff
@@ -13,6 +13,14 @@
       "description": "Allow multiple choices to be selected.",
       "default": false
     },
+    "placeholder": {
+      "type": "string",
+      "description": "Description of the input desired. Displayed when no choice has been selected.",
```
Comment (nesalang) [inline schemas/src/elements/inputs/Input.ChoiceSet.json]: nit: we could mention that we support place holder text only when isMultiSelect is set false.

Comment (andrewleader) [reply schemas/src/elements/inputs/Input.ChoiceSet.json]: Ah great catch Joseph!! Updated :)

**Comment by andrewleader** [comment]
Spec approved at the 7/12/2019 review meeting

**Comment by paulcam206** [comment]
closing out old PRs

**Pull request closed without merging**